include/mbedtls/ssl.h

```diff
@@ -4498,8 +4498,31 @@ void mbedtls_ssl_conf_session_tickets(mbedtls_ssl_config *conf, int use_tickets)
 
 #if defined(MBEDTLS_SSL_PROTO_TLS1_3)
 /**
- * \brief Enable / Disable TLS 1.3 handling of NewSessionTicket messages (client and TLS 1.3 only).
- *        (Default: MBEDTLS_SSL_ENABLE_NEW_SESSION_TICKETS_DISABLED)
+ * \brief Enable / Disable handling of TLS 1.3 NewSessionTicket messages (client and TLS 1.3 only).
+ *
+ *        The handling of TLS 1.3 NewSessionTicket messages is disabled by
+ *        default.
+ *
+ *        Contrary to TLS 1.2 tickets, the default value is disabled in Mbed TLS
+ *        3.6.x for backward compatibility with applications based on the Mbed TLS
+ *        client in the default configuration connecting to TLS 1.3 servers
+ *        supporting NewSessionTicket messages.
```

Comment:
Sorry, I don't understand this sentence, in particular "applications based on the Mbed TLS client".

Reply:
Suggestion: "Contrary to TLS 1.2 tickets, the default value is disabled in Mbed TLS 3.6.x for backward compatibility with client applications developed using Mbed TLS 3.5 or earlier with the default configuration." (then perhaps merge with the next paragraph).

Reply:
I mean an application that is using the Mbed TLS TLS client like ssl_client2 I'd say.